fastlane/actions/updraft.rb

```diff
@@ -6,21 +6,19 @@ module SharedValues
     class UpdraftAction < Action
       def self.run(config)
         begin
-        config[:ipa] = Actions.lane_context[SharedValues::IPA_OUTPUT_PATH] if Actions.lane_context[SharedValues::IPA_OUTPUT_PATH]
-
         UI.important("Collecting the necessary data to upload...")
 
         git_url = Helper.backticks("git remote get-url origin").to_s
         git_branch = Actions.git_branch.to_s
         git_tag = Helper.backticks("git tag -l --points-at HEAD").to_s
         git_commit_hash = Helper.backticks("git rev-parse HEAD").to_s
-        
+
         whats_new = Helper.backticks("git log -1 --pretty=%B").to_s
 
         bundle_version = Fastlane::Actions::GetIpaInfoPlistValueAction.run(ipa: config[:ipa], key: "CFBundleVersion").to_s
 
         build_type = other_action.is_ci? ? "CI" : "Fastlane"
-        
+
         curl_command = "curl -X PUT"
         curl_command << " -F 'app=@%s'" % config[:ipa]
```

Comment:
This parameter needs to be adjusted because the `config[:ipa]` value is not present for Android uploads using the `:apk` parameter.